Sources/Utilities/Storage/UserStorable.swift

```diff
@@ -21,22 +21,17 @@ extension UserStorable {
     }
     
     var validAuthenticatedUser: Bool {
-        guard let previouslyAuthenticatedUser else {
-            return false
-        }
+        guard let previouslyAuthenticatedUser else { return false }
```

Comment:
I think just 
`return previouslyAuthenticatedUser?.timeIntervalSinceNow.sign == .plus` 
will do the right thing and you don't need the guard statement